docs/developer-docs/web-apps/verifiable-credentials/issuer.mdx

```diff
@@ -0,0 +1,164 @@
+---
+keywords: [intermediate, tutorial, identity, credentials, verifiable credentials, issuer]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Become an issuer
+
+<MarkdownChipRow labels={["Intermediate", "Tutorial"]} />
+
+## Overview
+
+An **issuer** is a service or app that can issue a verifiable credential (VC) to a user. On ICP, an issuer is an exposed API that recieves calls from an identity provider and does not trigger any workflows itself. To become an issuer, your canister must implement the issuer API as described in [verifiable credential spec](https://github.com/dfinity/internet-identity/blob/main/docs/vc-spec.md).
+
+Issuers receive credentials requests from relying partys through an identity provider such as Internet Identity (II). The end-user must allow or revoke the communication before the credential is issued.
+
+Internet Identity is used as the identity provider in this tutorial.
+
+## Development of the issuer API
+
+The issuer API consists of four endpoints:
+
+1. [`vc_consent_message`](https://github.com/dfinity/internet-identity/blob/main/docs/vc-spec.md#1-consent-message): Responds with a message of consent to be displayed to the user to receive their consent on issuing the credentials.
+
+2. [`derivation_origin`](https://github.com/dfinity/internet-identity/blob/main/docs/vc-spec.md#2-derivation-origin): Returns the URL used to derive the user's principal for the issuer. 
+
+3. [`prepare_credential`](https://github.com/dfinity/internet-identity/blob/main/docs/vc-spec.md#3-prepare-credential): Check the validity of the request and prepare the actual credential requested.
+
+4. [`get_credential endpoint`](https://github.com/dfinity/internet-identity/blob/main/docs/vc-spec.md#4-get-credential): Issues the actual credential requested by the user.
+
+### Step 1: Obtain a consent message (`vc_consent_message`)
+
+First, the user must send a consent message to the issuer API endpoint `vc_consent_message` through the identity provider. A consent message is a text shown to the user to confirm that they consent to the issuing of a VC.
```

Comment:
```suggestion
First, the user must send a consent message request to the issuer API endpoint `vc_consent_message` through the identity provider. A consent message is a text shown to the user to confirm that they consent to the issuing of a VC.
```

Reply:
Not exactly. The user doesn't send a request. @jessiemongeon1 I believe we should change this paragraph.

> First, the identity provider sends a request to the issuer API to get the consent message to show the user. A consent message is a ...

Reply:
Revised per feedback